content/docs/integrations/travis-ci/index.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Travis CI
+lastUpdated: '2021-11-01T11:00:00.0Z'
+description: How to create Travis CI API Token for Travis CI plugin.
+---
+
+## Introduction
+
+In order to use the Travis CI plugin, Roadie needs an API token to communicate with Travis CI APIs.
+
+This is set within backstage at the following url:
+
+```text
+https://<tenant-name>.roadie.so/administration/settings/secrets
+```
+
+This page describes how to create and set up the Travis CI API.
+
+## Steps
+
+### Step 1: Create an API token
+
+Start by visiting the account settings page in [Travis](https://www.travis-ci.com/account/preferences).
+
+Click on the "Copy token" button under the API settings section.
+
+   ![Travis CI TOKEN](./copy-token.png)
+
+
+### Step 2: Store the token in Roadie
+Visit
+```text
+https://<tenant-name>.roadie.so/administration/settings/secrets
+``` and enter the token value from above into `TRAVISCI_AUTH_TOKEN`.
+
+
+## References
+
+- [Token, Token, Token](https://blog.travis-ci.com/2013-01-28-token-token-token)
```

Comment:
This part is not rendering properly. I think you need a new line after line 31 and on line 43 after the <pre>```</pre>


![travis](https://user-images.githubusercontent.com/736631/139813753-921c849c-05a3-44cc-b5ac-07cc7bb47530.png)